backend/clubs/views.py

```diff
@@ -4945,21 +4969,7 @@ def initiate_checkout(self, request, *args, **kwargs):
                 status=status.HTTP_403_FORBIDDEN,
             )
 
-        # Calculate cart total, applying group discounts where appropriate
-        ticket_type_counts = {
-            item["type"]: item["count"]
-            for item in cart.tickets.values("type").annotate(count=Count("type"))
-        }
-
-        cart_total = sum(
-            (
-                ticket.price * (1 - ticket.group_discount)
-                if ticket.group_size
-                and ticket_type_counts[ticket.type] >= ticket.group_size
-                else ticket.price
-            )
-            for ticket in tickets
-        )
```

Comment:
Was [previously broken](https://stackoverflow.com/questions/45768190/typeerror-manyrelatedmanager-object-is-not-iterable).